index.js

```diff
@@ -47,8 +47,8 @@ function initialize() {
     if (!psemver.satisfies(pkgJSON.engines.node)) {
       // TODO: Update this message when Node v12 is deprecated.
       message =
-        'New Relic for Node.js requires a version of Node equal to or\n' +
-        'greater than 12.0.0. Not starting!'
+        'New Relic for Node.js requires a version of Node >= 12.0.0.\n' +
```

Comment:
why not just include `${pkgJSON.engines.node}` so we don't have to change it and then can remove the TODO

Reply:
i can. was just trying to get a quick fix in place. i'll push a change in a few